.ci/build_wheels.sh

```diff
@@ -5,28 +5,12 @@ set -e -x
 
 # build based on python version from args
 PYTHON_VERSION="$1"
-case $PYTHON_VERSION in
-3.7)
-  PYBIN="/opt/python/cp37-cp37m/bin"
-  ;;
-3.8)
-  PYBIN="/opt/python/cp38-cp38/bin"
-  ;;
-3.9)
-  PYBIN="/opt/python/cp39-cp39/bin"
-  ;;
-3.10)
-  PYBIN="/opt/python/cp310-cp310/bin"
-  ;;
-3.11)
-  PYBIN="/opt/python/cp311-cp311/bin"
-  ;;
-esac
+PYBIN="/opt/python/cp${PYTHON_VERSION//.}-cp${PYTHON_VERSION//.}/bin"
 
 # build, don't install
 cd io
 "${PYBIN}/pip" install build
 "${PYBIN}/python" -m build --wheel
-auditwheel repair dist/ansys_mapdl_reader*.whl
+auditwheel repair dist/mapdl_archive*.whl
```

Comment:
```suggestion
auditwheel repair dist/ansys_mapdl_reader*.whl
```

The wheel name should be ``ansys_mapdl_reader`` if I am not mistaken. That's the package name.